app/lib/utils/file.js

```diff
@@ -0,0 +1,10 @@
+// Taken from https://github.com/kevva/executable and modified
```

Comment:
I take only the part that check the file mode, not the part that make the game.statSync (to not do it twice), and removed the part that check owner and group (maybe user wants to execute a command to discover he has no permissions)